.lycheeignore

```diff
@@ -1,11 +1,13 @@
 dev-example-okta.com
 .*-dsn.algolia.net
 www.linkedin.com/company/getunleash
+www.linkedin.com/in/*
```

Comment:
We have some linkedin urls because of contributor contact details. There get 999 error by linkedin, so adding to ignore.